src/components/Curators/MaintenanceRequests.vue

```diff
@@ -39,25 +39,25 @@
               {{ props.item.createdAt }}
             </td>
             <td>
+              <v-avatar
+                v-if="props.item.type"
+                class="mr-2"
+                :height="40"
+              >
+                <Icon
+                  :item="props.item.type"
+                  :height="40"
+                  wrapper-class=""
+                />
+              </v-avatar>
               <a :href="'#/' + props.item.id">
-                <span
-                  v-if="props.item.type"
-                  class="mr-2"
-                >
-                  <v-avatar
-                    v-if="Object.keys(recordType).includes(props.item.type)"
-                    size="38"
-                  >
-                    <img
-                      :src="'./' + recordType[props.item.type].icon"
-                    >
-                  </v-avatar>
-                </span>
                 {{ props.item.recordName }}
               </a>
             </td>
             <td>
-              {{ props.item.userNameID }}
+              <a :href="'#/users/' + props.item.userId">
```

Comment:
I'm not sure if the # is required here. Do you know, @terazus ?

Reply:
It is required as long as we are not in history mode.

Reply:
Are we to remain in history mode for deployment? If not then this will have to be reverted later.
If so then some will have to be added (see https://github.com/FAIRsharing/fairsharing.github.io/issues/804).

Reply:
We will have history mode in production, which means that we will need to removed all href '#'.
The easiest way i found to deal with this is to use a `goTo(link) ` method instead with `this.$router.push({path:link})` bound through a @click. Because $router already knows which we are on.

Reply:
If so then it might well be best to sort these links out now so we don't have to remember to come back to them later (hence #804).

Reply:
Well, I've been creating all the links in the curator panel in this way (e.g. https://github.com/FAIRsharing/fairsharing.github.io/blob/dev/src/views/Curators/Curator.vue#L68). Maybe I can create a ticket to modify them when real development happens.

Reply:
@ramGranell please create a ticket, then; I could approve this once I know the link issue has been noted.

Reply:
I created the ticket https://github.com/FAIRsharing/fairsharing.github.io/issues/805 but not sure if I explain it properly.